lib/anchor.dart

```diff
@@ -14,11 +14,11 @@ class Anchor {
   final double x;
   final double y;
 
-  Vector2 get relativePosition => Vector2(x, y);
+  Vector2 get toVector2 => Vector2(x, y);
```

Comment:
I think this should be a method to be consistent with others: toVector2()
or could asVector2 (getter)

Reply:
I'm ok with get relativePosition as well

Reply:
Good point, I'll go with `toVector2()` to just keep it the same all over the board.
I think `relativePosition` is quite a confusing term, "What is it in relation to, and what is the expected output?" is what I would ask myself without looking at the code.